tests/test_model_saving.py

```diff
@@ -859,5 +859,46 @@ def test_sequential_mxnet_model_saving_no_compile():
         save_mxnet_model(model, prefix='test', epoch=0)
 
 
+@pytest.mark.skipif((K.backend() != 'mxnet'),
+                    reason='Supported for MXNet backend only.')
+@keras_test
+def test_sequential_get_mxnet_model_info():
+    model = Sequential()
+    model.add(Dense(2, input_shape=(3,)))
+    model.add(RepeatVector(3))
+    model.add(TimeDistributed(Dense(3)))
+    model.compile(loss=losses.MSE,
+                  optimizer=optimizers.RMSprop(lr=0.0001),
+                  metrics=[metrics.categorical_accuracy],
+                  sample_weight_mode='temporal')
+    x = np.random.random((1, 3))
+    y = np.random.random((1, 3, 3))
+    model.train_on_batch(x, y)
+
+    data_names, data_shapes = K.get_mxnet_model_info(model)
```

Comment:
Can we also compare the data_names and data_shapes with K.save_mxnet_model() API? I assume it will be the same

Reply:
Good test case. Thanks for suggesting. Added.
Yes, they should be the same. if not, it is an issue. With your suggested testcase, we should catch that issue. Thanks.